### DISCUSSION TITLE
Is the code of site app.supabase.io open-sourced ?

### DISCUSSION THREAD

**Discussion by ImHype** [supabase/supabase#1270] [Questions] [answered]
I have a question. Is the code of site app.supabase.io not open-sourced ? I could not find any code of this site

**Comment by silentworks** [comment]
No, I don't think that code is open-sourced.

**Reply by ImHype** [reply]
Ok, Thanks

**Reply by MildTomato** [reply]
This is outdated. We have since open sourced it 🚀 
see https://github.com/supabase/supabase/discussions/1270#discussioncomment-3751890

**Comment by MildTomato** [comment, marked as answer]
yes, it is.

it can be found [here](https://github.com/supabase/supabase/tree/master/studio)
